### PR TITLE
Allow for populating note body when note is defined in frontmatter

### PR DESCRIPTION
**Pull request by enigmartyr**
Previously, only notes defined via HTML had their body populated via the `innerText`. These edits should allow for the frontmatter `description` property to achieve the same effect.

### CODE REVIEW

File: src/utils/events.ts [hunk 1]
```diff
@@ -96,6 +96,9 @@ export const getEventData = (
   const noteTitle      = retrieveEventValue(
     eventObject, 'title', file.name.replace( '.md', '' ), frontMatterKeys?.titleKey
   )
+  const noteBody       = retrieveEventValue( 
+    eventObject, 'description', isHTMLElementType(eventObject) ? event.innerText : ''
```
Comment (seanlowe) [inline src/utils/events.ts:100]: `event.innerText` needs to be `eventObject.innerText` in this context.

Also, the defaultValue param is a little messy, can you move it to it's own variable (perhaps above where `color` is declared?) for readability?

**Review by seanlowe — CHANGES_REQUESTED** [review]
Hey, looks good overall. Just a few notes and then we can get this merged in.

1. Small mishap on `events.ts`
2. Please write some "release notes" in the README regarding this change (release for this will be 2.1.11) and move the current release notes to the changelog.
3. After doing that, run `npm run bump:patch` to update the version numbers on everything (manifest.json, package.json, & package-lock.json)
4. This is a particularity of mine but please alphabetize `noteBody` with the other `note..` variables/keys
5. I actually recently _deleted_ the description attribute from the "Insert HTML event" function `createTimelineEventInCurrentNote` so you'll want to add that back into `commands.ts`: `newEventElement.setAttribute( 'data-description', '' )`
6. and lastly, if you want (optional here - I can do it later), you can add a small section on the Description attribute to the docs in the `01_html_arguments.md` file.

Thanks!

File: src/utils/events.ts (hunk 1 above)
Comment (enigmartyr) [reply src/utils/events.ts:100]: Yes, I'll have to do a proper clone but after I get a working dev environment I can take care of that. Previously I was just tweaking things in the raw plugin js file and then I forked the repo and was manually editing files in the Github's web based texy editor to reflect the changes I made in my local js file. Admittedly sloppy

**Review by seanlowe — APPROVED** [review]
Looks good!

**Comment by enigmartyr** [comment]
> Looks good!

I actually hadn't ran npm run bump:patch yet because I wanted to build the plugin and test it and make sure it worked as intended first but getting the build script to run proved difficult for me

**Comment by seanlowe** [comment]
If you would open another issue for tracking down the build issues, I believe that would be extremely helpful for anyone who may want to contribute in the future.

**Comment by enigmartyr** [comment]
image
image
image
to tell the truth, these are the changes I made to get it to build but I hesitate to say they're the correct changes since I hardly know what I've done

**Pull request merged**